spec/execution_tests.lua

```diff
@@ -1425,6 +1425,166 @@ function execution_tests.run(compile_file, backend, _ENV, only_compile)
         end)
     end)
 
+    -- Note: Implementation currenly only supports float and uses fabs().
+    -- Test avoids differences that may be caused by needing integers.
+    describe("math.abs builtin", function()
+        compile([[
+            function m.absolute_value(x: float): float
+                return math.abs(x)
+            end
+        ]])
+
+        it("works on positive numbers", function()
+            run_test([[
+                assert(1.0 == test.absolute_value(1.0))
+                assert(7.0 == test.absolute_value(7.0))
+            ]])
+        end)
+
+        it("works on negative numbers", function()
+            run_test([[
+                assert(1.0 == test.absolute_value(-1.0))
+                assert(11.0 == test.absolute_value(-11.0))
+            ]])
+        end)
+
+        it("returns NaN on NaN", function()
+            run_test([[
+                local x = test.absolute_value(0.0 / 0.0)
+                assert(x ~= x)
+            ]])
+        end)
+    end)
+
+    describe("math.exp builtin", function()
+        compile([[
+            function m.exponential_value(x: float): float
+                return math.exp(x)
+            end
+        ]])
+
+        it("works on positive numbers", function()
+            run_test([[
+                assert(1.0 == test.exponential_value(0.0))
+                assert(2.718 == tonumber(string.format("%.3f", 
+                    test.exponential_value(1.0))))
+            ]])
+        end)
+
+        it("works on negative numbers", function()
+            run_test([[
+                assert(0.368 == tonumber(string.format("%.3f", 
+                    test.exponential_value(-1.0))))
```

Comment:
Maybe this fits in a single line? We use a 100-column lines instead of 80-columns. A good way to get this working is to install an editorconfig plugin, see the editorconfig file for details.